public/pages/editor/wizard.html

```diff
@@ -57,6 +57,8 @@
             <label>Number of results</label>
           </div>
           <div class="col-md-6">
```

Comment:
It will be nice to have operator select and threshold value text input field aligned in one line, for example:
```
Number of results     [ Is above]    [ 100 ]
```
To achive this, you can use the following markup instead of the current `<div class="col=md-6">` block
```
<div class="col-md-2">
  <select
    class="form-control"
    name="operator"
    ng-model="watcher.$$condition_operator"
    ng-options="operator.operator as operator.name for operator in watcher.$$condition_operators"
  >
  </select>
</div>
<div class="col-md-4">
  <input type="number" min="0" class="form-control" id="watcherTitle" ng-model="watcher.$$condition_value">           
</div>
```

Reply:
I agree. Trying your approach now.